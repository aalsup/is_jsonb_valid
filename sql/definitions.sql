-- valid definition
-- valid definition schema
SELECT is_jsonb_valid('{"$ref":"http://json-schema.org/draft-04/schema#"}', '{"definitions":{"foo":{"type":"integer"}}}');
-- invalid definition
-- invalid definition schema
SELECT is_jsonb_valid('{"$ref":"http://json-schema.org/draft-04/schema#"}', '{"definitions":{"foo":{"type":1}}}');