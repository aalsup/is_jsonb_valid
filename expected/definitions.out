-- valid definition
-- valid definition schema
SELECT is_jsonb_valid('{"$ref":"http://json-schema.org/draft-04/schema#"}', '{"definitions":{"foo":{"type":"integer"}}}');
 is_jsonb_valid 
----------------
 t
(1 row)

-- invalid definition
-- invalid definition schema
SELECT is_jsonb_valid('{"$ref":"http://json-schema.org/draft-04/schema#"}', '{"definitions":{"foo":{"type":1}}}');
 is_jsonb_valid 
----------------
 f
(1 row)

